test/e2e/operator_test.go

```diff
@@ -773,176 +942,6 @@ func TestDNSOverTLSToleratesMissingSourceCM(t *testing.T) {
-	}
-	coreDNSPods := &corev1.PodList{}
-	if err := cl.List(context.TODO(), coreDNSPods, client.MatchingLabelsSelector{Selector: selector}, client.InNamespace(dnsDaemonSet.Namespace)); err != nil {
-		t.Fatalf("failed to list pods for dns daemonset %s/%s: %v", dnsDaemonSet.Namespace, dnsDaemonSet.Name, err)
-	}
-	catCmd := []string{"cat", "/etc/coredns/Corefile"}
-
-	// Get the openshift-cli image.
-	var (
-		cliImage      string
-		cliImageFound bool
-	)
-	for _, ver := range co.Status.Versions {
-		if ver.Name == statuscontroller.OpenshiftCLIVersionName {
-			if len(ver.Version) == 0 {
-				break
-			}
-			cliImage = ver.Version
-			cliImageFound = true
-			break
-		}
-	}
-	if !cliImageFound {
-		t.Fatalf("failed to find the %s version for clusteroperator %s", statuscontroller.OpenshiftCLIVersionName, co.Name)
-	}
-
-	// Create the client Pod.
-	testClientForDNSLogging := buildPod("test-client-dnslogging", upstreamPodNs, cliImage, []string{"sleep", "3600"})
-	if err := cl.Create(context.TODO(), testClientForDNSLogging); err != nil {
-		t.Fatalf("failed to create pod %s/%s: %v", testClientForDNSLogging.Namespace, testClientForDNSLogging.Name, err)
-	}
-	defer func() {
-		if err := cl.Delete(context.TODO(), testClientForDNSLogging); err != nil {
-			t.Fatalf("failed to delete pod %s/%s: %v", testClientForDNSLogging.Namespace, testClientForDNSLogging.Name, err)
-		}
-	}()
-	// Wait for the client Pod to be ready.
-	name := types.NamespacedName{Namespace: testClientForDNSLogging.Namespace, Name: testClientForDNSLogging.Name}
-	err = wait.PollImmediate(1*time.Second, 60*time.Second, func() (bool, error) {
-		if err := cl.Get(context.TODO(), name, testClientForDNSLogging); err != nil {
-			t.Logf("failed to get pod %s/%s: %v", name.Namespace, name.Name, err)
-			return false, nil
-		}
-		for _, cond := range testClientForDNSLogging.Status.Conditions {
-			if cond.Type == corev1.ContainersReady &&
-				cond.Status == corev1.ConditionTrue {
-				return true, nil
-			}
-		}
-		return false, nil
-	})
-	if err != nil {
-		t.Fatalf("failed to observe ContainersReady condition for pod %s/%s: %v", testClientForDNSLogging.Namespace, testClientForDNSLogging.Name, err)
-	}
-
-	found := 0
-	for _, corednspod := range coreDNSPods.Items {
-
-		// Dig the example dns forwarding host.
-		digCmd := []string{"dig", "test.svc.cluster.local"}
-		if err := lookForStringInPodExec(testClientForDNSLogging.Namespace, testClientForDNSLogging.Name, testClientForDNSLogging.Name, digCmd, "NXDOMAIN", 2*time.Minute); err != nil {
-			t.Fatalf("failed to dig %v", err)
-		}
-
-		if err := lookForStringInPodExec(corednspod.Namespace, corednspod.Name, "dns", catCmd, "class denial error", 2*time.Minute); err != nil {
-			t.Fatalf("failed to set Debug logLevel for operator %s: %v", opName, err)
-		}
-
-		// Get the CoreDNS image used by the test upstream resolver.
-		co := &configv1.ClusterOperator{}
```

Comment:
This is removed in the "moved version" of the test. It didn't seem like this was necessary and added a considerable amount of time to the test.